examples/interfaces/stan.ipynb

```diff
@@ -0,0 +1,992 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Interfacing with Stan\n",
+    "This notebook explains how it is possible to interface with [Stan](https://mc-stan.org/) to make use of their awesome library for log-probability calculations, gradient calculation (via autodiff) and their large zoo of probability distributions.\n",
```

Comment:
Can we reword this to reflect the actual code more? I think we can only interact with a log pdf at the moment?